<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>240</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="seven_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
	background-color: white;
}
QPushButton {
	background-color: silver;
	border: 1px solid gray;
	border-radius: 0px;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="eight_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
	background-color: white;
}
QPushButton {
	background-color: silver;
	border: 1px solid gray;
	border-radius: 0px;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="nine_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
	background-color: white;
}
QPushButton {
	background-color: silver;
	border: 1px solid gray;
	border-radius: 0px;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="six_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
	background-color: white;
}
QPushButton {
	background-color: silver;
	border: 1px solid gray;
	border-radius: 0px;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="five_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
	background-color: white;
}
QPushButton {
	background-color: silver;
	border: 1px solid gray;
	border-radius: 0px;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="four_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
	background-color: white;
}
QPushButton {
	background-color: silver;
	border: 1px solid gray;
	border-radius: 0px;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="three_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
	background-color: white;
}
QPushButton {
	background-color: silver;
	border: 1px solid gray;
	border-radius: 0px;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="two_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
	background-color: white;
}
QPushButton {
	background-color: silver;
	border: 1px solid gray;
	border-radius: 0px;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="one_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
	background-color: white;
}
QPushButton {
	background-color: silver;
	border: 1px solid gray;
	border-radius: 0px;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="calc_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
	background-color: silver;
}
QPushButton {
	background-color: gray;
	border: 1px solid black;
	border-radius: 0px;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="zero_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
	background-color: white;
}
QPushButton {
	background-color: silver;
	border: 1px solid gray;
	border-radius: 0px;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="dot_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
	background-color: silver;
}
QPushButton {
	background-color: gray;
	border: 1px solid black;
	border-radius: 0px;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="sub_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>192</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
	background-color: white;
}
QPushButton {
	background-color: orange;
	border: 1px solid #ff5500;
	border-radius: 0px;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>144</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
	background-color: white;
}
QPushButton {
	background-color: orange;
	border: 1px solid #ff5500;
	border-radius: 0px;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mult_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>96</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
	background-color: white;
}
QPushButton {
	background-color: orange;
	border: 1px solid #ff5500;
	border-radius: 0px;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divide_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>48</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
	background-color: white;
}
QPushButton {
	background-color: orange;
	border: 1px solid #ff5500;
	border-radius: 0px;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
	background-color: white;
}
QPushButton {
	background-color: orange;
	border: 1px solid #ff5500;
	border-radius: 0px;
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLabel" name="output">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>120</width>
      <height>240</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>240</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>240</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
